src/SAHB.GraphQL.Client.Subscription/GraphQLSubscriptionWebSocketClient.cs

```diff
@@ -31,6 +31,17 @@ public GraphQLSubscriptionWebSocketClient(IGraphQLFieldBuilder fieldBuilder, IGr
             this.deserialization = deserialization;
```

Comment:
You can avoid duplicated code in the constructors by using constructor chaining. For example line 25 - 32 can be replaced by:
`
public GraphQLSubscriptionWebSocketClient(IGraphQLFieldBuilder fieldBuilder, IGraphQLQueryGeneratorFromFields queryGenerator, IGraphQLDeserialization deserialization, CancellationToken cancellationToken)
   : this(new ClientWebSocket(), fieldBuilder, queryGenerator, deserilization, cancellationToken)
{ }
`

Reply:
> You can avoid duplicated code in the constructors by using constructor chaining. For example line 25 - 32 can be replaced by:
> `public GraphQLSubscriptionWebSocketClient(IGraphQLFieldBuilder fieldBuilder, IGraphQLQueryGeneratorFromFields queryGenerator, IGraphQLDeserialization deserialization, CancellationToken cancellationToken) : this(new ClientWebSocket(), fieldBuilder, queryGenerator, deserilization, cancellationToken) { }`

That 'inheritance' [?] was ny first approach, but it didn't seem to work - hence replicating most of the same code in the two constructors instead.